hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -202,6 +202,7 @@ static const struct quirk xpadneo_quirks[] = {
 	DEVICE_OUI_QUIRK("E4:17:D8",
 			 XPADNEO_QUIRK_NO_PULSE | XPADNEO_QUIRK_NO_TRIGGER_RUMBLE |
 			 XPADNEO_QUIRK_NO_MOTOR_MASK | XPADNEO_QUIRK_NINTENDO),
+	DEVICE_OUI_QUIRK("44:16:22", XPADNEO_QUIRK_SHARE_BUTTON),
```

Comment:
Does the gamepad have rumble motors in the triggers? If not, we may want to set the no-trigger-rumble quirk, too.

Reply:
Is there a way to test if it does?

Reply:
You'd either need a screwdriver, or you use the hidraw rumble tester from the misc folder.

Reply:
How do I get a hidraw node? With this module loaded or unloaded, I don't see a new /dev/hidraw* node.

Reply:
Ah, it's `hidraw0`. Yes, it does have trigger rumbles.